app/consumer/ante_handler.go

```diff
@@ -15,7 +15,7 @@ import (
 type HandlerOptions struct {
 	ante.HandlerOptions
 
-	IBCChannelkeeper channelkeeper.Keeper
+	IBCChannelKeeper *ibckeeper.Keeper
```

Comment:
Should we call this `IBCKeeper` instead of  `IBCChannelKeeper` for more clarity?